documentation/adr/0003-use-mqtt-to-publish-trip-modifications.md

```diff
@@ -0,0 +1,26 @@
+# 3. Use MQTT to publish Trip Modifications
+
+Date: 2024-07-04
+
+## Status
+
+Proposed
+
+## Context
+
+We need to send [GTFS Trip Modifications](https://gtfs.org/realtime/feed-entities/trip-modifications/)
+to Transit Data.
+
+## Decision
+
+To minimize the amount of resources used and latency between Skate publishing or
+un-publishing a Trip Modification, we are going to use MQTT to form a 
+Event Stream between Skate and Transit Data.
+
+Since this is an output of Skate, we will not make this service required to do
+local development.
+
+## Consequences
+
+We'll be adding a new technology and service to Skate, this is more surface area
```

Comment:
```suggestion
We'll be adding a new technology and service to Skate, so there will be more surface area
```

OR

```suggestion
Since we'll be adding a new technology and service to Skate, there will be more surface area
```

Tiny grammar nit.